src/plugins/intel_cpu/src/nodes/pad.cpp

```diff
@@ -254,19 +232,17 @@ void Pad::PadExecutor::paramsInitialization(const PadAttrs& attrs,
     const auto& srcDims = srcBlockMemDesc->getBlockDims();
     const auto& dstDims = dstBlockMemDesc->getBlockDims();
 
-    params.attrs.prc = srcMemPtr->getDesc().getPrecision();
     params.srcDims = srcDims;
     params.dstDims = dstDims;
+    params.attrs.prc = srcMemPtr->getDesc().getPrecision();
     params.dataSize = params.attrs.prc.size();
 
     auto fillingInParameters =
-        [&](std::vector<unsigned int>& parameter, const size_t type, const size_t size, const int value) {
+        [&](std::vector<int>& parameter, const size_t type, const size_t size, const int value) {
```

Comment:
Since `VectorIdx` is now std::vector<int32_t>, we could use int32_t to replace all int in `fillingInParameters`

Reply:
I renamed to `VectorIdx`. Thanks